fineract-loan/src/main/java/org/apache/fineract/cob/data/LoanDataForExternalTransfer.java

```diff
@@ -31,4 +32,13 @@ public class LoanDataForExternalTransfer {
     Integer loanStatus;
```

Comment:
I think we can have the enum here. It is only used internally.

Reply:
I wasn't sure that there were points where we actually transfer this object or parts of it to external systems so I decided to err on the safer side. I will change the property.